static/src/js/components/PermissionForm/PermissionForm.jsx

```diff
@@ -220,7 +220,10 @@ const PermissionForm = () => {
   const { data } = useSuspenseQuery(GET_COLLECTION_FOR_PERMISSION_FORM, {
     skip: conceptId === 'new',
     variables: {
-      conceptId
+      conceptId,
+      params: {
+        limit: 2000
```

Comment:
so this is 2000, but the others are 100?

Reply:
Yeah this was the query that resulted in the corrupting ACLs.  before it only pulled in 20 and then if the user saved, it would only save 20 even though they had 100s in the database

Reply:
answered in person